src/main/lua/exasolrls/rls_adapter.lua

```diff
@@ -49,7 +49,23 @@ function M.drop_virtual_schema()
     return {type = "dropVirtualSchema"}
 end
 
-function M.refresh()
+---
+-- Refresh the metadata of the Virtual Schema.
+-- <p>
+-- Re-reads the structure and data types of the schema protected by RLS.
+-- </p>
+--
+-- @param exa_metadata Exasol metadata (not used)
```

Comment:
```suggestion
-- @param _ Exasol metadata (not used)
```
the parameter name is `_` (only mentioned once)

Reply:
Fixed, good find!